variables.tf

```diff
@@ -13,3 +13,35 @@ variable "aws_profile" {
   type        = string
   description = "The profile that was used to deploy this module. If the default profile / default credentials are used, seet this value to null."
 }
+
+variable "api_token" {
+  type        = string
+  description = "This is your Bridgecrew platform Api token Set as and Environment variable TF_VAR_api_token"
+}
+
+variable "template_version" {
```

Comment:
I don't think this should be writeable. If they specify anything other than the expected value, stuff will break.

Reply:
yeah, i know that now :(